pkg/storage/cloudimpl/cloudimpltests/gcs_storage_test.go

```diff
@@ -161,3 +161,37 @@ func TestFileDoesNotExist(t *testing.T) {
 		require.True(t, errors.Is(err, cloudimpl.ErrFileDoesNotExist))
 	}
 }
+
+func TestCompressedGCS(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	user := security.RootUserName()
+	ctx := context.Background()
+
+	// gsutil cp /usr/share/dict/words gs://cockroach-fixtures/words-compressed.txt
+	gsFile1 := "gs://cockroach-fixtures/words.txt?AUTH=implicit"
```

Comment:
do we need to skip this test if we're running w/out implicit authentication configured?

Reply:
ugh, yes. Thanks!